icu4c/source/i18n/unicode/listformatter.h

```diff
@@ -65,6 +65,13 @@ struct ListFormatData : public UMemory {
 /**
  * An immutable class containing the result of a list formatting operation.
  *
+ * When calling nextPosition():
+ * The fields are returned from left to right. The special field category
```

Comment:
"left to right" is problematic if the text is right to left. In order from the beginning, or something.